Formula/fdroidcl.rb

```diff
@@ -0,0 +1,24 @@
+class Fdroidcl < Formula
+  desc "F-Droid desktop client"
+  homepage "https://github.com/mvdan/fdroidcl"
+  url "https://github.com/mvdan/fdroidcl/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "532a8c4c93216cbf13378ff409c06a08d48e8baee6119a50ed43dc0ce9ec7879"
+  license "BSD-3-Clause"
+  head "https://github.com/mvdan/fdroidcl.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build"
+    bin.install "fdroidcl"
+  end
+
+  test do
+    system bin/"fdroidcl", "update"
```

Comment:
If this test isn't good enough, please tell me and I'll improve it.

Reply:
Is there anything we can check in the output here?

Reply:
It outputs this (if the index was downloaded before and it's up to date it says "not modified" instead of "done"):
```
Downloading https://f-droid.org/repo/index-v1.jar... done
```

It may output other lines if there are other F-Droid repos added in the config file, or output nothing if (for whatever reason) the config file lists no repos (note that the F-Droid repo is enabled by default if the config file is not present).

Reply:
Any suggestions on what I should do here?

Reply:
Something like
```ruby
assert_match "done", shell_output("#{bin}/fdroidcl update")
```
might work here.

Reply:
As I stated before, the problem is:

> [the command] output nothing if (for whatever reason) the config file lists no repos (note that the F-Droid repo is enabled by default if the config file is not present).

I admit though that I can't imagine a case where somebody would do that, so maybe it's fine if we disregard that possibility. Thoughts?

Reply:
Where does it look for a config file? The test is meant to be run in CI, so we should be able to control what is and isn't in the config file. We can write one to `testpath` if we have to.

Reply:
Ah, is that so? Then there's no need to worry; it has the F-Droid repo enabled by default when the config file is not defined.

Reply:
@carlocab implemented your suggestion.